Tests/SentryTests/SentryHttpStatusCodeRangeTests.swift

```diff
@@ -0,0 +1,46 @@
+import XCTest
+
+class SentryHttpStatusCodeRangeTests: XCTestCase {
+    
+    func testWithinRange() {
+        let range = HttpStatusCodeRange(min: 500, max: 599)
+        
+        XCTAssertTrue(range.is(inRange: 550))
+    }
+    
+    func testMinWithinRange() {
+        let range = HttpStatusCodeRange(min: 500, max: 599)
+        
+        XCTAssertTrue(range.is(inRange: 500))
+    }
+    
+    func testLowerMinNotWithinRange() {
+        let range = HttpStatusCodeRange(min: 500, max: 599)
+        
+        XCTAssertFalse(range.is(inRange: 499))
+    }
+    
+    func testMaxWithinRange() {
+        let range = HttpStatusCodeRange(min: 500, max: 599)
+        
+        XCTAssertTrue(range.is(inRange: 599))
+    }
+    
+    func testHigherMaxNotWithinRange() {
+        let range = HttpStatusCodeRange(min: 500, max: 599)
+        
+        XCTAssertFalse(range.is(inRange: 600))
+    }
+    
+    func testStatusCodeWithinRange() {
+        let range = HttpStatusCodeRange(statusCode: 500)
+        
+        XCTAssertTrue(range.is(inRange: 500))
+    }
+    
+    func testStatusCodeNotWithinRange() {
+        let range = HttpStatusCodeRange(statusCode: 500)
+        
+        XCTAssertFalse(range.is(inRange: 200))
```

Comment:
`l`: Would improve the quality of the test a bit.
```suggestion
        XCTAssertFalse(range.is(inRange: 499))
```

Reply:
There are tests covering the corner/edge cases such as 499 and 600, IMO its easier to read 200 than 499 since 200 is OK and it should not be within the range.
499 depends if you know what the default is, one may assume 4xx is also errored.